lib/resources.yml

```diff
@@ -63,11 +63,13 @@ js:
       - angular2: '*'
     React (latest):
       - react: '*'
+      - react-dom: '*'
 
   sourceFormat:
     options:
       - ES5
       - ES6
+      - JSX+ES6
```

Comment:
This option feels awkward. Questions:
- Does JSX belong to a framework (React), or will it be used beyond that? 
- Can I use JSX with ES5, ES6 or TypeScript?
- Should we add a separate question about using JSX?